src/repository/transform_photos.py

```diff
@@ -0,0 +1,181 @@
+from __future__ import annotations
+
+from typing import Type
+
+import cloudinary
+from cloudinary import uploader
+from fastapi import HTTPException, status
+from sqlalchemy.orm import Session
+import uuid
+
+from src.conf.config import settings
+from src.database.models import Photo
+from src.repository.photos import get_photo_by_photo_id
+from src.schemas import TransformPhotoModel
+
+
+async def update_orig_photo_with_transformed_photo(
+    db: Session, photo_id: int, photo_url: str, updated_by: int, photo_description: str
+):
+    """
+    Method updates the original photo in the db with a transformed photo.
+
+    :param db: DB instance.
+    :type db: Session.
+    :param photo_id: Original photo identifier.
+    :type photo_id: int.
+    :param photo_url: Original photo URL.
+    :type photo_url: str.
+    :param updated_by: User who updated a photo.
+    :type updated_by: int.
+    :param photo_description: Photo description.
+    :type photo_description: str.
+    :return: Photo instance.
+    :rtype: Photo.
+    """
+    original_photo: Type[Photo] | None = await get_photo_by_photo_id(
+        db=db, photo_id=photo_id
+    )
+    if not original_photo:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Photo with id {photo_id} wasn't found",
+        )
+    original_photo.url = photo_url
+    original_photo.updated_by = updated_by
+    original_photo.is_transformed = True
+    original_photo.description = photo_description
+    db.add(original_photo)
+    db.commit()
+    db.refresh(original_photo)
+    return original_photo
+
+
+async def create_transformed_photo_in_db(
+    db: Session,
+    orig_photo_id: int,
+    photo_url: str,
+    updated_by: int,
+    photo_description: str,
+) -> Photo:
+    """
+    Method that creates the new row in the DB table public.photos for transformed photo.
+
+    :param db: DB instance.
+    :type db: Session.
+    :param orig_photo_id: Original photo identifier.
+    :type orig_photo_id: int.
+    :param photo_url: Original photo URL.
+    :type photo_url: str.
+    :param updated_by: User who updated a photo.
+    :type updated_by: int.
+    :param photo_description: Photo description.
+    :type photo_description: str.
+    :return: Photo instance.
+    :rtype: Photo.
+    """
+    new_photo = Photo(
```

Comment:
Suggestion to rename 'new_photo' to 'transformed_photo' \ 'which_transformed_photo' which is more selfdocumenting and clear.